easybuild/easyconfigs/f/fosscuda/fosscuda-2019b.eb

```diff
@@ -18,9 +18,8 @@ local_comp_mpi_tc = ('gompic', version)
 # We need GCC, CUDA and OpenMPI as explicit dependencies instead of
 # gompic toolchain because of toolchain preperation functions.
 dependencies = [
-    ('GCC', local_gccver),  # part of gompic
+    ('gcccuda', version),  # part of gompic
```

Comment:
This is a tricky change as I've seen EB complaining (for another hierarchy) that GCC and CUDA *have* to be there in the toolchain modulefile; try building something with this fosscuda, and also check subtoolchains.

Reply:
You're right...

Reply:
And it bloody well says so in the comment above... brain not fully engaged...